src/app_config.cpp

```diff
@@ -156,8 +156,8 @@ ConfigOpt *opts[] =
 // OCPP 1.6 Settings
   new ConfigOptDefenition<String>(ocpp_server, "", "ocpp_server", "ows"),
   new ConfigOptDefenition<String>(ocpp_chargeBoxId, "", "ocpp_chargeBoxId", "cid"),
-  new ConfigOptDefenition<String>(ocpp_idTag, "", "ocpp_idTag", "idt"),
-  new ConfigOptDefenition<String>(tx_start_point, "tx_pending", "tx_start_point", "otx"),
+  new ConfigOptDefenition<String>(ocpp_authkey, "", "ocpp_authkey", "oky"),
+  new ConfigOptDefenition<String>(ocpp_idtag, "", "ocpp_idtag", "oid"),
```

Comment:
Should probably keep the `idt` short key to persist any saved confign in existing devices